tests/integration/compose/docker_compose_iceberg_lakekeeper_catalog.yml

```
@@ -0,0 +1,119 @@
+services:
+  lakekeeper:
+    image: quay.io/lakekeeper/catalog:latest-main
```

```suggestion
    image: quay.io/lakekeeper/catalog:latest
```
We should use the latest flag (latest release) instead of nightly builds for more stability.